README.md

```diff
@@ -1,70 +1,122 @@
 # 96Boards.org
 
-The 96Boards website is built using the Jekyll static site generator. It is based off the jumbo-jekyll-theme used for all of the Linaro static sites. With the move from Wordpress to Static we have introduced GitHub/Community driven content that allows 96Boards.org users to submit issues about any of the pages on 96Boards.org through the associated Git respository. This GitHub repository is combined with the 96Boards Documentation repo to build the site. Documentation is copied to the `_documentation` collection upon site builds with [bamboo.linaro.org](https://bamboo.linaro.org). 
+![Build Status](https://bamboo.linaro.org/plugins/servlet/wittified/build-status/BOARDS-BUILDDEV0)
 
-![Build Status](https://bamboo.linaro.org/plugins/servlet/wittified/build-status/BOARDS-BUILDDEV0) 
+The 96Boards website is built using the [Jekyll](https://jekyllrb.com/) static site generator. It uses the Linaro [jumbo-jekyll-theme](https://github.com/linaro-marketing/jumbo-jekyll-theme) (which is used for all of the Linaro static websites). With the source for the source for the 96Boards.org website available on GitHub in this repo, it makes it easy for community/technical contributors to submit content. The 96Boards documentation repo is synced into the _documentation directory during site builds that occur on our Atlassian Bamboo CI instance.
+
+## Contributions
+
+We happy to consider any contributions/feature requests that you may have. Please submit a PR with your changes and we will take a look. You can also use the `Github Edit` buttons available on all the website pages to help locate the file you wish to edit/raise and issue about.
+
+### Simple edits on GitHub
+
+A simple edit to content can be made directly on the GitHub website. Find the page you would like to make changes to and click the "pencil" icon.
+
+![Simple GitHub changes - pencil edit](/assets/images/content/pencil_edit.png)
+
+Once you've made your change you'll be able to add a change title and description. Make sure to select the "Create a new branch for this commit and start a pull request" option.
+
+![Simple GitHub changes - change text and description](/assets/images/content/create_new_branch.png)
+
+Then click "Propose file change" button. Maintainers of this website repo will then review and publish your change accordingly.
+
+If you'd like a more detailed guide on making changes on GitHub then view the GitHub documentation [here](https://help.github.com/en/github/managing-files-in-a-repository/editing-files-in-your-repository).
+
+### Using a the git CLI
+
+If you're familiar with using the git cli then you can simply fork this repository and clone your fork. Then submit pull request on a `new` branch based off of the `develop` branch. This is due to the 96Boards.org website repo following the [Git flow](https://danielkummer.github.io/git-flow-cheatsheet/) methodology. Push your changes to your fork and then create a pull request via the GitHub UI.
```

Comment:
"you can simply fork this repository and clone your fork" ... fork and clone? Surely just fork?

Reply:
If you're using the Git CLI then how will the repo be available to you by simply forking the repo?